pandas/_libs/tslibs/period.pyx

```diff
@@ -163,6 +111,31 @@ cdef extern from *:
     int FR_UND
 
 
+cdef int max_value(int left, int right) nogil:
+    if left > right:
+        return left
+    return right
+
+
+cdef int min_value(int left, int right) nogil:
+    if left < right:
+        return left
+    return right
+
+
+cdef int64_t get_daytime_conversion_factor(int from_index, int to_index) nogil:
+    cdef:
+        int row = min_value(from_index, to_index)
+        int col = max_value(from_index, to_index)
+    # row or col < 6 means frequency strictly lower than Daily, which
+    # do not use daytime_conversion_factors
+    if row < 6:
```

Comment:
can u use the enumerated here?

Reply:
huh?  This is just moving the existing C code into cython.

Reply:
I get it, but the 6 is pretty opaque